File: ProgramOne/code.py
def fibonacci(n):
    fib = [0,1]
    for i in range(2,n+1):
        fib.append(fib[i-1] + fib[i-2])
    return fib

print(fibonacci(3))
print(fibonacci(5))
print(fibonacci(7))
print(fibonacci(9))

File: ProgramOne/code.js
function fibonacci(num) {
    const fib = [0,1]
    for (let i = 2; i <= num; i++) {
        fib[i] = fib[i-1] + fib[i-2]
    }
    return fib.toString()
}

console.log(fibonacci(3));
console.log(fibonacci(5));
console.log(fibonacci(7));
console.log(fibonacci(9));